src/models/uses.rs

```diff
@@ -143,6 +147,18 @@ pub(crate) trait RepositoryUsesExt {
 }
 
 impl RepositoryUsesExt for RepositoryUses {
+    fn path_without_ref(&self) -> String {
+        format!(
+            "{}/{}{}",
+            self.owner,
+            self.repo,
+            self.subpath
+                .as_ref()
+                .map_or("".to_owned(), |subpath| format!("/{subpath}"))
+        )
+        .to_lowercase()
```

Comment:
Performed lowercasing here to be consistent with the other methods which seem to treat this case insensitively as well.